apps/base/utils.py

```diff
@@ -156,7 +156,7 @@ def get_boto3_client(resource, aws_keys):
 
 def get_or_create_sqs_queue_object(queue_name):
     if settings.DEBUG or settings.TEST:
-        queue_name = "evalai_submission_queue"
+        # queue_name = "evalai_submission_queue"
```

Comment:
Why do we need to comment this line?

Reply:
On dev end, if we use same queue name for testing, both the submission worker and unit tests would monitor & poll the same queue for messages. So, this would cause problems because the results would be unexpected.

Reply:
I don't think this will create problems (unless forced) as while testing you will explicitly send the messages into the queue and then test it.
One case I can think of is that when you run tests and at the same time create submissions from UI and it will create conflicts. Other than this do you have any other cases in your mind?

Reply:
If the submission worker is working in the background on the same queue, it will poll the message regardless if it’s from the UI or from the unit tests so I think it could fail. I’ll try uncommenting though since it affects only the dev environment

Reply:
This is what I wrote above, no?

Reply:
Yes. But, since there are two consumers (the unit test & submission queue) for one producer (the unit test), the message can be picked up by any of the 2 causing unexpected results. This will happen only on dev env, since that's the only place we're overriding the queue name